app/util/data_preparation/api/bitbucket_clients.py

```diff
@@ -57,7 +64,7 @@ def get_projects(self, max_results=500):
 
     def get_users(self, name_filter, max_results=500):
         return self.get_entities(entity_name='users',
-                                 filter=name_filter,
+                                 filter_=name_filter,
```

Comment:
I updated the name of this parameter since it hides built-in function `filter`: https://docs.python.org/2/library/functions.html
According to this doc: https://dbader.org/blog/meaning-of-underscores-in-python there is a convention to add underscore after the name in this case.